examples/Dicom/src/parseDicomFiles.js

```diff
@@ -182,13 +179,23 @@ const parseDicomFiles = async (fileList) => {
     patient.parseMetaData(dicomMetaData, file)
   }
 
+  // Set up promises
+  const parseFiles = [...fileList].map((file) => {
+    const promise = parseFile(file)
+    if (skipOnFailure) {
+      promise.catch((error) => {
+        failures.push({ file, error })
+      })
```

Comment:
Even with the `catch` not throwing again (just storing the failure), I get an uncaught exception when calling [`Promise.all`](https://github.com/InsightSoftwareConsortium/itk-js/pull/252/commits/c41ac4cc26c5caf8b1eaf6d1c2ff2e4d1b124677#diff-62f35c82e16d848a11710ce525c9a4faR196)...

Reply:
I think that's by design, does it work if you wrap up the Promise.all with try/catch in that case?

Reply:
Thanks. Addressed in 1aa0c8807a0b68968a30a3d50df0535c5c7b37e8.